test/shaping/data/text-rendering-tests/tests/SHLANA-1.tests

```diff
@@ -0,0 +1,52 @@
+../fonts/TestShapeLana.ttf:--font-size=1000 --ned --remove-default-ignorables --font-funcs=ft:U+1A20,U+1A6B:[uni1A20|uni1A6B@762,0]
```

Comment:
Why all these tests are using `ft` font functions?

Reply:
Some where ok with ot callbacks also and some weren't, not sure why, needs more investigation.

Reply:
I see, thanks.

Reply:
Might be because we didn't have glyph names implemented in ot-font back when I integrated this test suite.  So, please someone investigate and report. :)